lib/Runtime/Base/ScriptContext.cpp

```diff
@@ -2051,6 +2051,14 @@ namespace Js
                 indirectFunctionTables[sigId][i] = localModuleFunctions[funcIndex];
             }
 
+            uint32 startFunc = wasmModule->info->GetStartFunction();
+            if (startFunc != -1)
+            {
+                Js::Var v = localModuleFunctions[startFunc];
+                PropertyRecord const * mainProp = nullptr;
+                GetOrAddPropertyRecord(_u("main"), lstrlen(_u("main")), &mainProp);
```

Comment:
is this true? I don't think we should be adding this onto export object. we need only call it.